File: Codeforces/C++/problemset/Way Too Long Words.cpp
// Problem Link: https://codeforces.com/problemset/problem/71/A

#include <iostream>
#include <string>
using namespace std;
int main()
{
    long long n;
    string s;
    cin >> n;
    for(int i = 0; i < n; i++)
    {
        cin >> s;
        if(s.size() > 10)
            cout << s[0] << s.size() - 2 << s[s.size() - 1] << "\n";
        else
            cout << s << "\n";
    }

    return 0;
}

File: Codeforces/Kotlin/src/main/kotlin/problems/WayTooLongWords.kt
package problems.problems

/* Problem Link: https://codeforces.com/problemset/problem/71/A */

fun main() {
    val n = readln().toInt()

    for (test in 1..n) {
        val word = readln()

        if (word.length > 10) {
            println("${word.first()}${word.length - 2}${word.last()}")
        } else {
            println(word)
        }
    }
}